formicaio/umbrel-app.yml

```diff
@@ -0,0 +1,29 @@
+manifestVersion: 1
+id: formicaio
+name: Formicaio
+tagline: Il potere delle formiche
+icon: ""
+category: social
```

Comment:
Would this not fit better into the **bitcoin** or **network** section?

Reply:
Agreed, I'll set it to bitcoin even that currently it's a bit more related to p2p networks and Ethereum tokens.

Reply:
...I actually set it to 'networking' for now, until a category like 'etherum' or 'p2p networks' is available.